org.cubocore.CoreGarage.yml

```diff
@@ -0,0 +1,55 @@
+app-id: org.cubocore.CoreGarage
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: coregarage
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home   # Required to archive any files and open any archived files
+  - --filesystem=xdg-config:create
+  - --device=dri
+  - --talk-name=org.freedesktop.Notifications
+cleanup:
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+  - /share/pkgconfig
+  - /man
+  - /share/man
+  - /lib/*.la
+  - /lib/*.a
+modules:
+  - name: libcprime
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: git
+        url: https://gitlab.com/cubocore/libcprime.git
+        tag: v4.5.0
+  - name: libarchive-qt
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: git
+        url: https://gitlab.com/marcusbritanicus/libarchive-qt
+        tag: v2.0.4
+  - name: coregarage
+    post-install:
+      - install -Dp -m 644 org.cubocore.CoreGarage.appdata.xml /app/share/appdata/org.cubocore.CoreGarage.appdata.xml
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCMAKE_LIBRARY_PATH=/app/lib
```

Comment:
I don't think this is needed either.

Reply:
we need those arg to build with cmake properly, cmake doesn't initialize if build type/lib path/prefix is not set.

Reply:
prefix will be set.